### DISCUSSION TITLE
sjtureport 英文环境下的首段缩进、字体设置以及类似 sjtuthesis 的 titlepage

### DISCUSSION THREAD

**Discussion by Shimushushushu** [sjtug/SJTUThesis#1002] [提问] [answered]
使用 `sjtureport 2023/09/25 v2.0.3`，以及 `lang=en` 的选项下，章节和小节的首段有缩进，请问这是正确的行为吗？如果我希望令首段缩进消失的话，除了手动加 `\noindent` 应该怎么处理？我参考了 https://github.com/sjtug/SJTUThesis/discussions/766 以及 https://github.com/sjtug/SJTUThesis/discussions/604 ，但是加上 `\ctexset{autoindent = false}` 之后没有反应，而目前的 `sjtureport` 里面也没有 `indent-headings` 这个选项。

MWE:
```
\documentclass[lang=en, cjk-font=none, text-font=libertinus]{sjtureport}

% \ctexset{
%   autoindent=false,
% }

\sjtusetup{
  style={
    % indent-headings=false,
  },
}

\usepackage{lipsum}

\begin{document}
\chapter{Foo}
\lipsum
\end{document}
```

同时，`\documentclass` 那行是我的第二个问题：虽然设置了 `lang=en`，但是中文字体还是会默认被加载，这会导致无法用 `pdflatex` 编译文件。相对地，`SJTUBeamer` 则没有这个问题。这是出于某种考量的正确的行为吗？

最后，我看到 https://github.com/sjtug/SJTUTeX/issues/20 有提到 `sjtureport` 沿用 `sjtuthesis` 的封面，请问近期有计划实现吗？

**Comment by AlexaraWu** [comment, marked as answer]
1. 没有什么特殊的考量，`sjtureport` 和 `sjtuarticle` 只不过是 `sjtuthesis` 的衍生产品，所以沿用了一些 thesis 的设置；首段缩进相关的 CTeX 选项是 `afterindent`，详见 `ctex` 文档；
2.  `sjtureport` 实质上是一个定制版的 `ctexrep`，无论什么语言选项都是带中文支持的；SJTUBeamer 并不是文档类，而是样式包；
3.  `sjtureport` 没有计划制作内置的封面。

**Reply by Shimushushushu** [reply]
谢谢，打算滚回去用旧版本了 :sob:
